src/main/java/minicraft/entity/mob/Sheep.java

```diff
@@ -46,7 +45,14 @@ public void render(Screen screen) {
 	@Override
 	public void tick() {
 		super.tick();
-		if (age - ageWhenCut > WOOL_GROW_TIME) cut = false;
+		if (random.nextInt(1000) == 0) { // Grazing
+			Tile tile = level.getTile(x >> 4, y >> 4);
+			// If tall grasses are present, these are consumed and then turn into grass tiles.
+			if (tile instanceof GrassTile) {
```

Comment:
Shouldn't sheep eat tall grass? Sheep can't eat grass that is only a couple of centimeters long.

You can use classes in the other pull request, and write that that PR is required before this one.

Reply:
This is just a game, so we could just assert that sheep could eat any height of grass.

Reply:
Yes, but what is the point of having tall grass then?

Reply:
First, we can have more fancy visual effect on plane grass tiles. Second, the sources and chances of getting wheat seeds and grass seeds are separated as dedicated sources.

Reply:
I don't understand what is so problematic with making sheep eat tall grass. The comment even says it does such.

Reply:
Sheep would eat both tall grass and grass tiles.

Reply:
I'll accept that.